src/Java.Interop.Tools.JavaSource/Java.Interop.Tools.JavaSource/SourceJavadocToXmldocGrammar.BlockTagsBnfTerms.cs

```diff
@@ -78,7 +80,7 @@ internal void CreateRules (SourceJavadocToXmldocGrammar grammar)
 					parseNode.AstNode   = p;
 				};
 
-				var nonSpaceTerm = new RegexBasedTerminal ("[^ ]", "[^ ]+") {
+				var nonSpaceTerm = new RegexBasedTerminal ("[^ \n\r]", "[^ \n\r]+") {
```

Comment:
Should this also include `\t`?

Reply:
I didn't see an instance where this was required but I think it makes sense, let me try to test it quickly.